app/src/main/AndroidManifest.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.findmyrecycling">
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Add Internet permission in the manifests file.
https://www.geeksforgeeks.org/retrofit-with-kotlin-coroutine-in-android/